class UserExistsException(BaseException):
    def __init__(self, message):
        super(message)